src/V3EmitCFunc.h

```diff
@@ -680,7 +680,10 @@ class EmitCFunc VL_NOT_FINAL : public EmitCConstInit {
         iterateAndNextConstNull(nodep->exprp());
         puts("}\n");
     }
-    void visit(AstNodeCase* nodep) override {  // LCOV_EXCL_LINE
+    void visit(AstNodeGen* nodep) override {  // LCOV_EXCL_LINE
+        nodep->v3fatalSrc("Generate constructs should have been reduced out");
+    }
```

Comment:
Should be able to be much earlier, somewhere after V3Begin, maybe V3LinkDotScope?